.github/workflows/upload-assets.yaml

```diff
@@ -34,12 +34,16 @@ jobs:
           fi
       - id: 'auth'
         name: 'Authenticate to Google Cloud'
-        if: ${{ steps.build.outputs.UPLOAD_ASSETS }}
         uses: 'google-github-actions/auth@v1'
+        if: ${{ steps.build.outputs.UPLOAD_ASSETS == 'true' }}
         with:
           workload_identity_provider: ${{ secrets.WORKLOAD_IDENTITY_PROVIDER }}
-          service_account: ${{ secrets.WORKLOAD_SERVICE_ACCOUNT }}
-      - id: 'upload'
-        if: ${{ steps.build.outputs.UPLOAD_ASSETS }}
-        run: ./weave/frontend/bundle.sh
-
+          service_account: ${{ secrets.WORKLOAD_IDENTITY_SERVICE_ACCOUNT }}
+      - id: 'upload-and-push'
+        if: ${{ steps.build.outputs.UPLOAD_ASSETS == 'true' }}
+        run: |
+          ./weave/frontend/bundle.sh
+          git config --global user.name 'Weave Build Bot'
+          git config --global user.email weave@wandb.com
+          git commit -a -m 'chore(bot): update frontend bundle sha'
+          git push
```

Comment:
I think there's a built in action step that will update instead of this, but if this works 👍

Reply:
hmm, I did a quick google search and didn't see any official ones. I saw some third party ones but I was skeptical. If you got one in mind, let me know